css/common.css

```diff
@@ -0,0 +1,52 @@
+:root {
+  --gray: #ededee;
+  --red: #a05941;
+  --black: #133032;
+  --white: #fff;
+  --transwhite: rgba(0, 0, 0, 0.75);
+}
+* {
+  padding: 0;
+  margin: 0;
+  border: 0;
+  box-sizing: border-box;
+  outline: 0;
+}
+body {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
```

Comment:
Sorry I forgot to delete that because I used to give min-height to Main then I removed that and did this to avoid any mess up later. I'll remove that . Thanks!